tests/conftest.py

```diff
@@ -327,6 +334,20 @@ def _make_package(name, version="0.1", install_requires=None, extras_require=Non
         # Create a README to avoid setuptools warnings.
         (package_dir / "README").touch()
 
+        # Create a module to make the package importable.
+        (package_dir / name).with_suffix(".py").touch()
+
+        if build_system_requires:
+            with (package_dir / "pyproject.toml").open("w") as fp:
+                fp.write(
```

Comment:
FYI pathlib has a nicer API for this:
```suggestion
            (package_dir / "pyproject.toml").write_text(
```
+ we should always specify `encoding="utf-8"`. I think it emits a warning on newer Pythons, otherwise.

Reply:
You are correct but don't you think it is more important to be consistent?

Reply:
Yep, consistency is a valid reason to skip doing it in this particular PR. It's probably better to change it across the code base separately.
Marking this as resolved.